web/src/message_events.ts

```diff
@@ -421,10 +447,15 @@ export function update_messages(events) {
                 any_message_content_edited = true;
 
                 // Update raw_content, so that editing a few times in a row is fast.
-                anchor_message.raw_content = event.content;
+                if (event.content !== undefined) {
+                    anchor_message.raw_content = event.content;
+                } else {
+                    delete anchor_message.raw_content;
+                }
```

Comment:
Or just change `raw_content?: string` to `raw_content?: string | undefined` in `type Message`; that’s generally fine as long as nobody’s checking `"raw_content" in message` or `Object.hasOwn(message, "raw_content")`.

Reply:
I thought we were trying to avoid things that were both optional and explicitly maybe `undefined`?

Reply:
I went ahead and did this